cmd/multiFaScan/multiFaScan.go

```diff
@@ -0,0 +1,66 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
+
+// Scan multiple Fasta alignment for a user-specified pattern (N for now) and report bed regions in reference sequence coordinates
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/fasta"
+	"log"
+)
+
+func multiFaScan(s Settings) {
+	aln := fasta.Read(s.InFile)
+	answerBedPos := fasta.ScanN(aln, s.QueryName)
+	var currentBed bed.Bed
+	var ans []bed.Bed
+	for i := 0; i < len(answerBedPos); i++ {
+		currentBed = bed.Bed{Chrom: s.Chrom, ChromStart: answerBedPos[i][0], ChromEnd: answerBedPos[i][1], Name: aln[0].Name, FieldsInitialized: 4} // Name field is reference species name
+		ans = append(ans, currentBed)
+	}
+	bed.Write(s.OutFile, ans)
+}
+
+func usage() {
+	fmt.Print(
+		"multiFaScan - Scan multiple Fasta alignment for a user-specified pattern (N for now) and report bed regions in reference sequence coordinates.\n" +
```

Comment:
Is the intended use to only find Ns, or will you eventually try and find other DNA motifs in multiFa?